eventstore/models.py

```diff
@@ -1094,3 +1094,39 @@ def clean(self):
                 date(self.dob_year, self.dob_month, self.dob_day)
             except ValueError as e:
                 raise ValidationError(f"Invalid date of birth date, {str(e)}")
+
+
+class OpenHIMQueue(models.Model):
+    class ObjectType:
+        PREBIRTH_REGISTRATION = "PrebirthRegistration"
+        CHW_REGISTRATION = "CHWRegistration"
+        PUBLIC_REGISTRATION = "PublicRegistration"
+        CHANNEL_SWITCH = "ChannelSwitch"
+        OPTOUT = "OptOut"
+        choices = (
+            (PREBIRTH_REGISTRATION, "PrebirthRegistration"),
+            (CHW_REGISTRATION, "CHWRegistration"),
+            (PUBLIC_REGISTRATION, "PublicRegistration"),
+            (CHANNEL_SWITCH, "ChannelSwitch"),
+            (OPTOUT, "OptOut"),
```

Comment:
Not a big thing, but I think we usually use the snake case for these values, eg. `prebirth_registration`, for the value that gets stored in the database, and then title case for the values that get shown in the UI, eg. `Prebirth Registration`

Reply:
Thanks for the review, yes that makes sense, I initially wanted to get the models dymanically so tried to keep them the same as the class name